examples/lfric/eg17/full_example_extract/Makefile

```diff
@@ -1,7 +1,7 @@
 # -----------------------------------------------------------------------------
 # BSD 3-Clause License
 #
-# Copyright (c) 2020, Science and Technology Facilities Council.
+# Copyright (c) 2020-2021, Science and Technology Facilities Council.
```

Comment:
I have the same Intel compilation problems with this example as with the `full_example` above.

Reply:
As above.